content/en/getting_started/synthetics/browser_test.md

```diff
@@ -61,31 +64,25 @@ You can specify the number of consecutive failures that should happen before con
 Retry test 2 times after 300 ms in case of failure
 ```
 
-
 You can also configure your test to only trigger a notification when your application goes down for a certain amount of time and number of locations. In the below example, the alerting rule is set to send a notification if the test fails for three minutes on two different locations:
 
-
 ```text
 An alert is triggered if your test fails for 3 minutes from any 2 of 13 locations
 ```
 
 #### Notify your team
 
-
-Add an alert name to the **Monitor Name** field and write a message for the alert. You can use [integrations][4] such as Slack, PagerDuty, and webhooks to route your alert to specific services and teams.
-
+Add an alert name to the **Monitor Name** field and write a message for the alert. You can use [integrations][5] such as Slack, PagerDuty, and webhooks to route your alert to specific services and teams.
 
 You can set your alert notification to re-notify if the alert has not been resolved and define the priority of the alert, ranging from **P5 (Info)** to **P1 (Critical)**.
 
-{{< img src="getting_started/synthetics/configured-browser-test.gif" alt="Configured browser test"  style="width:90%;">}}
+{{< img src="getting_started/synthetics/configured-browser-test.mp4" alt="Example browser test configuration" video="true"  >}}
```

Comment:
The video file name is spelled correctly here but the actual file name has a misspelling.

Change:
`static/images/getting_started/synthetics/configured-brower-test.mp4`
to
`static/images/getting_started/synthetics/configured-browser-test.mp4`

Reply:
🙌🏻🙌🏻